.gitignore

```diff
@@ -57,4 +57,5 @@ backend/core/local_models/*
 
 
 ## scripts
-.migration_info
\ No newline at end of file
+.migration_info
+package-lock.json
```

Comment:
Should not be added to the .gitignore

Reply:
Hey, Zineb! How are you doing? But .migration_info does have private info from db connection (Supabase) and Package-lock.json is auto generated when npm install is run. Is it really needed to remove these lines from .gitignore? Thank you. :)

Reply:
Was talking about the package-lock.json. It has been created because you used npm instead of yarn. If you insist on using npm, it's fine, you would just need to manually remove the package-lock.json from your committed files before creating your commit

Reply:
Oh, got it. How could I start using yarn instead of npm? I am not very experienced with this. Thank you for your help. :)